src/trade/liquidation.ts

```diff
@@ -29,13 +35,68 @@ export const getLiquidationPrice = (
     initialAccFees,
     context
   );
-  const liqPriceDistance =
+  const liqThresholdP = getLiqPnlThresholdP(
+    context.liquidationParams,
+    trade.leverage,
+    context.contractsVersion
+  );
+
+  let liqPriceDistance =
     (trade.openPrice *
-      (trade.collateralAmount * 0.9 - (borrowingFee + closingFee))) /
+      (trade.collateralAmount * liqThresholdP - (borrowingFee + closingFee))) /
     trade.collateralAmount /
     trade.leverage;
 
+  if (
+    context?.contractsVersion === ContractsVersion.V9_2 &&
```

Comment:
Important to always use >= ContractsVersion.V9_2 so it works for next versions as well